packages/fastify-vite/config.js

```diff
@@ -44,7 +44,7 @@ const DefaultConfig = {
     if (!clientModule) {
       return null
     }
-    const { routes, ...others } = clientModule
+    let { routes, ...others } = clientModule
     if (typeof routes === 'function') {
```

Comment:
tbh, polymorphic reassignment is a messy thing, maybe we can rewrite this to be immutable?

Reply:
something like?

```js
const routes = typeof clietModule.routes === 'function' 
  ? await clientModule.routes
   : clientModule.routes
return Object.assign({}, clientModule, { routes })
```

Reply:
I like it, used that with a small fix — thanks @Uzlopak @kibertoad @Fdawgs